content/pages/board.md

```diff
@@ -8,3 +8,4 @@ The following is a list of the current NBLUG board members.
 * **Treasurer:** Matt Dasilva
 * **Scribe:** Glenn Kerbein
 * **Directors at Large:** Tom Most and John Morio Sakaguchi
+* **Secretary** Robert Thille
```

Comment:
Heck if I know :-)  I just show up to the meetings.  I don't think I've actually done anything 'official' since I was elected :-O